scss/widgets/generic/radioButton/_index.scss

```diff
@@ -101,11 +101,3 @@ $generic-radiobutton-inner-size: $generic-radiobutton-size - $generic-radiobutto
     }
   }
 }
-
-.dx-rtl .dx-radiobutton,
-.dx-rtl.dx-radiobutton {
-  &.dx-radiobutton-checked .dx-radiobutton-icon-dot {
-    margin-right: math.div($generic-radiobutton-inner-size, 4) + $generic-radiobutton-border-width;
```

Comment:
`$generic-radiobutton-inner-size` variable is not using anymore

Reply:
fixed